### PR TITLE
av-12/0 Privacy Page + av-10/0 Terms and Policy

### PR DESCRIPTION
**Pull request by jib-data**

### CODE REVIEW

File: assets/js/main
Comment (jizzel) [inline assets/js/main:1]: Shto eta?

**Review by jizzel — CHANGES_REQUESTED** [review]
Make sure your branch is up to date with the main branch.

**Comment by jib-data** [comment]
Change file to from Terms and Conditions to Terms and Policy

File: terms.html [hunk 1]
```diff
@@ -0,0 +1,394 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta content="width=device-width, initial-scale=1.0" name="viewport" />
+
+    <title>Terms and Conditions</title>
```
Comment (jizzel) [inline terms.html:7]: Update the title accordingly.

**Pull request merged**